### PR TITLE
Static / Dynamic Nodes

### PR DESCRIPTION
**Pull request by imadtassaoui**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,8 +41,8 @@
     "immer": "^10.1.1",
     "lodash": "^4.17.21",
     "lucide-react": "^0.445.0",
-    "rc-virtual-list": "^3.14.5",
```
Comment (Aceix) [inline package.json:42]: why are we removing this? i think it was used somewhere in a virtual table/list component

File: src/components/modals/add-node-modal/AddNodeDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,751 @@
...
+  const [downloadLink, setDownloadLink] = useState<string>('');
+  const [linuxInstallCode, setLinuxInstallCode] = useState<string>('');
+
+  const antdTheme = useTheme();
+  const appleStoreLink = 'https://apps.apple.com/us/app/netmaker-rac/id6479694220?itsct=apps_box_badge&amp;itscg=30200';
+  const playStoreLink =
+    'https://play.google.com/store/apps/details?id=com.net.netmaker&pli=1&utm_source=nmui&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1';
+
+  const updateLinuxInstallCode = (version: string) => {
+    const baseUrl = 'https://fileserver.netmaker.io/releases/download/v0.25.0';
```
Comment (Aceix) [inline src/components/modals/add-node-modal/AddNodeDialog.tsx:100]: let's put all links in the constant file

Comment (Aceix) [inline src/components/modals/add-node-modal/AddNodeDialog.tsx:100]: change to `https://fileserver.netmaker.io/releases/download/${store.serverConfig?.Version}`

File: src/components/modals/add-node-modal/AddNodeDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,751 @@
...
+    const newCode = `wget -O netclient ${baseUrl}/netclient-linux-${version} && chmod +x ./netclient && sudo ./netclient install`;
+    setLinuxInstallCode(newCode);
+  };
+
+  const getDownloadLink = (instruction: any, selectedVersion: string) => {
```
Comment (Aceix) [inline src/components/modals/add-node-modal/AddNodeDialog.tsx:105]: make this a useCallback else it'll cause unnecessary re-renders in L160's useEffect

File: src/components/modals/add-node-modal/AddNodeDialog.tsx [hunk 4]
```diff
@@ -0,0 +1,751 @@
...
+        <h3 className="text-sm-semibold">Manage Users</h3>
+      </div>
+      <p className="text-text-secondary">To manage users you can do it from User Management.</p>
+      <Link to={'/users'} target="_blank">
+        <button className="flex items-center justify-center gap-2 px-4 py-2 rounded text-text-secondary bg-button-plain-fill-default">
+          <UsersIcon className="w-4 h-4" />
+          <span className="text-sm-semibold">Manage Users</span>
+          <ArrowTopRightOnSquareIcon className="w-4 h-4" />
+        </button>
+      </Link>
```
Comment (Aceix) [inline src/components/modals/add-node-modal/AddNodeDialog.tsx:46]: would it be better to rather open in the same tab?

**Review by Aceix — CHANGES_REQUESTED** [review]
image

increase the z-index of the popover. should be over the buttons

File: package.json (hunk 1 above)
Comment (imadtassaoui) [reply package.json:42]: I think it's still there just got reordered somehow

**Review by Aceix — APPROVED** [review]

**Pull request merged**